src/formatDate.js

```diff
@@ -50,7 +50,54 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dateArr = date.split(fromFormat[3]);
+  let year = '';
+  let month = '';
+  let day = '';
+
+  for (const i of fromFormat) {
```

Comment:
Use a more descriptive variable name, instead of 'i' use 'element' or 'formatElement'.